backend/src/main/java/com/kinsella/kinsellaBackend/controllers/ProductsController.java

```diff
@@ -29,17 +38,66 @@ public Collection<Products> getProductByID(@PathVariable Long productID) {
         return productsRepository.findByProductID(productID);
     }
 
+    /*
+    @GetMapping({"/{ticketId}/comment/{commentNumber}", ""})
+    public ResponseEntity getTicketComment(@PathVariable(name = "ticketId") Long ticketId, @PathVariable(name = "commentNumber") Long commentNum){
+    */
+
+    @PostMapping(path = "/admin/add/{name}/{price}/{stock}")
```

Comment:
For POST requests the convention is to add the requests parameters in the body rather than in the URL - you should just be able to append your params to the `formData` object on the client and pull them out here so that everything is transferred in the same way.

Reply:
Applied this change 👍